relentless/potential/potential.py

```diff
@@ -529,7 +530,44 @@ def __next__(self):
         return next(self.coeff)
 
 class Tabulator:
+    """Creates an object that organizes multiple potential functions together.
+
+    Bins accumulated energy and force values for multiple potential functions,
```

Comment:
Bins -> evaluates throughout this code. ("Binning" is used for histogramming. You are just evaluating at a set of r values.)